frontend/src/routes/Applications/Overview.tsx

```diff
@@ -1041,7 +1041,9 @@ export default function ApplicationsOverview() {
             action={
               <>
                 {appCreationButton}
-                <TextContent>{viewDocumentation(DOC_LINKS.MANAGE_APPLICATIONS, t)}</TextContent>
+                <div>
+                  <ViewDocumentationLink doclink={DOC_LINKS.MANAGE_APPLICATIONS} />
+                </div>
```

Comment:
With the current changes, I'm not sure why you would need the new `<div>` as `<TextContent>` was moved inside `<ViewDocumentationLink>`, so the resulting DOM tree should be the same?